app/javascript/modules/listings/DirectoryHelpers.tsx

```diff
@@ -185,14 +185,16 @@ export const getListingCards = (listings, directoryType, stackedDataFxn, hasFilt
     return (
       <ListingCard
         key={index}
+        stackedTable={true}
```

Comment:
this is probably just a convention nitpick, but I'm used to passing fixed boolean props with presence/absence of the attribute. does TypeScript change this though? 🤔

Reply:
Just a personal preference, I don't mind either way, but this seems to be how we're doing it elsewhere - super down to update that style